inventory/templates/inventory/catalogue_item_info_view.html

```diff
@@ -0,0 +1,171 @@
+        <thead>
+            <tr>
+                <th scope="col">Owned by</th>
+                <th scope="col">Note</th>
+                <th scope="col">Added</th>
+                <th scope="col">Actions</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for link in active_links %}
+            <tr>
+                <td>{{ link.owner }}</td>
+                <td>{{ link.note }}</td>
+                <td>{{ link.added_since }} by {{ link.added_by.get_full_name }}</td>
+                <td>
+                    <div class="btn-group" role="group" aria-label="Basic example">
+                        <a class="btn btn-secondary"
+                           href="{% url "inventory:catalogue_item_links" type_id=item_type item_id=item.id link_id=link.id %}"
+
+                        >Edit</a>
+                        <form method="post" id="deactivate-{{ link.id }}"
+                              action="{% url "inventory:catalogue_item_link_deactivation" type_id=item_type item_id=item.id link_id=link.id %}">
+                            {% csrf_token %}
+                        </form>
+                        <input type="submit" form="deactivate-{{ link.id }}"
+                               class="btn btn-warning" value="Deactivate"></button>
+                    </div>
+                </td>
+
+            </tr>
+        {% endfor %}
+
+
+        </tbody>
+    </table>
+
+    <hr>
+    <h3>Previously at the Knights</h3>
+
+
+    <table class="table table-striped table-hover">
+        <thead>
+        <tr>
+            <th scope="col">Owned by </th>
+            <th scope="col">Note</th>
+            <th scope="col">Removed on</th>
+            <th scope="col">Actions</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for link in inactive_links %}
+            <tr>
+                <td>{{ link.owner }}</td>
+                <td>{{ link.note }}</td>
+                <td>{{ link.added_since }}</td>
+                <td>
+                    <div class="btn-group" role="group" aria-label="Basic example">
+                        <a class="btn btn-secondary"
+                           href="{% url "inventory:catalogue_item_links" type_id=item_type item_id=item.id link_id=link.id %}"
+
+                        >Edit</a>
+                        <form method="post" id="activate-{{ link.id }}"
+                              action="{% url "inventory:catalogue_item_link_activation" type_id=item_type item_id=item.id link_id=link.id %}">
+                            {% csrf_token %}
+                        </form>
+                        <input type="submit" form="activate-{{ link.id }}"
+                               class="btn btn-warning" value="Activate">
+                        <a href="{% url "inventory:catalogue_item_link_deletion" type_id=item_type item_id=item.id link_id=link.id %}"
+                           class="btn btn-danger">
+                            Delete
```

Comment:
There's a bit of a mismatch(?) between where delete buttons are located: for boardgames they're in the edit page, but for ownerships they're only on the overview page.
I suggest we streamline this and move this "Delete' button also to the edit page. For active items, we can disable this button and add a tooltip stating that only inactive items can be deleted. I don't think such a note is anywhere at this point (or I completely missed it :P), but it might not be obvious to users that they cannot delete active items

Reply:
Good point. I removed item removal from the edit page and to the maintenance page. Makes no sense to be able to remove it without knowing whom are the owners anyway